app/models/Prepaid.coffee

```diff
@@ -43,6 +44,16 @@ module.exports = class Prepaid extends CocoModel
       
     return 'available'
 
+  courseType: ->
```

Comment:
Not really the `courseType` so much as the `licenseType` or `typeDescription`, or something like this, should rename this function for clarity.